### PR TITLE
OOP school library: add basic UI

### PR DESCRIPTION
**Pull request by anthonymr**
## Milestone IV: add basic UI

In this PR I Create `App` interface class and an interactive CLI menu in the `main.rb` file. 

The App interface is able to:
- List all books
- List all people
- Create a person (teacher or student, not a plain Person)
- Create a book
- Create a rental
- List all rentals for a given person's ID

In the CLI menu, the user is able to:
- Create a new person (student or teacher)
- Create a new book
- Create a new rental entry
- List all books
- List all persons
- List all rental entries by person's ID

ℹ️ To use this library:

1. Clone the repo by running `git@github.com:anthonymr/school-library.git`
2. Move to the new folder by running `cd school-library`
3. Execute the program by running `ruby main.rb`

### CODE REVIEW

File: main.rb [hunk 1]
```diff
@@ -1,64 +1,112 @@
...
+  date = gets.chomp
+
+  app.create_rental(date, book_index, people_index)
+end
+
+def option6(app)
+  print 'ID of person: '
+  id = gets.chomp.to_i
+  app.list_rentals_for(id)
+end
```
Comment (DammyShittu) [inline main.rb:70]: - [ ] All app logic should be moved to the `app.rb` file. Kindly move these code to the `app.rb` file. The purpose of the `main.rb` file is mainly for listing the initial options and taking in inouts.

**Review by DammyShittu — CHANGES_REQUESTED** [review]
Hi @anthonymr,

## Highlights
✔️ Basic UI matches the video requirement

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: main.rb (hunk 1 above)
Comment (anthonymr) [reply main.rb:70]: Hi @DammyShittu, thanks for your time. If you can check those methods (option#) the only thing they do is print messages on the CLI and read data from inputs, after that, just call an App class method.

Comment (peterrobert) [reply main.rb:70]: @anthonymr I checked the implementation and it's true. The logic is in the app file. The methods in the main file are just presentations methods

**Review by peterrobert — APPROVED** [review]
Hi @anthonymr , 


### Highlights!

[ ] Good feature branch name ✔️
[ ] Descriptive pull request ✔️


Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread or tag me @peterrobert if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: main.rb (hunk 1 above)
Comment (anthonymr) [reply main.rb:70]: Hi @peterrobert, thanks for the clarification.

**Pull request merged**